src/components/data_submitter/DataAccessGovernance.js

```diff
@@ -0,0 +1,168 @@
+    newConsentGroups[idx] = undefined;
+    setConsentGroups(newConsentGroups);
+  };
+
+  const updateConsentGroup = (idx, update) => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups[idx] = {
+      ...consentGroups[idx],
+      ...update,
+    };
+    updateFormData({
+      consentGroups: newConsentGroups,
+    });
+  };
+
+
+
+  return div({}, [
+    div({}, [
+
+      // open or closed access
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === OPEN_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: OPEN_ACCESS,
+          }),
+          description: 'Open Access',
+        }),
+      ]),
+
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: CLOSED_ACCESS,
+          }),
+          description: 'Closed Access',
+        }),
+      ]),
+
+      
+      div({
+        isRendered: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+      },
+        [
+          // select dac
+
+          h(SearchSelect, {
+            id: 'data_submission_select_dac',
+            name: 'data_submission_select_dac',
+            onSelection: (selection) => updateFormData({
+              dataAccessCommitteeId: selection,
+            }),
+            options: dacs.map((dac) => {
+              return { key: dac.dacId, displayText: dac.name };
+            }),
+            placeholder: 'Search for institution...',
+            value: formData.dataAccessCommitteeId,
+          }),
+
+
+          // add consent groupa
+          div({className: 'right-header-section', style: {
+            display: 'flex',
+            alignItems: 'flex-end'
+          }}, [
+            a({
+              id: 'btn_addInstitution',
+              className: 'btn-primary btn-add common-background',
+              onClick: () => addNewConsentGroup(),
+            }, [
+              span({}, ['Add Consent Group'])
+            ])
+          ]),
+
+          // consent groups
+          consentGroups
+          .map((group, idx) => {
+            if (isNil(group)) {
+              return div({}, []);
+            } 
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
            }
```